safekeeper/src/pull_timeline.rs

```diff
@@ -401,7 +401,10 @@ pub async fn handle_request(
         request.timeline_id,
     ));
     if existing_tli.is_ok() {
-        bail!("Timeline {} already exists", request.timeline_id);
+        info!("Timeline {} already exists", request.timeline_id);
+        return Ok(PullTimelineResponse {
+            safekeeper_host: None,
```

Comment:
FYI: This is a preliminary check, but we may encounter `timeline already exists` in another place: in `global_timelines.load_temp_timeline()` below. It may happen if we call `pull_timeline` two times in parallel.
It's not clear how to handle it correctly, though: it may also throw an error "Creating", that means another request is also in the process of creating timeline, but it's not done yet and there is no guarantee of success

TLDR: maybe not worth fixing now